react/src/Analyses/index.tsx

```diff
@@ -264,98 +364,7 @@ export default function Analyses() {
-                                                    : event.target.value
-                                            )
-                                        }
-                                        fullWidth
-                                    >
-                                        {enums?.PriorityType.map(p => (
-                                            <MenuItem key={p} value={p}>
-                                                {p}
-                                            </MenuItem>
-                                        ))}
-                                        <MenuItem value="None">None</MenuItem>
-                                    </Select>
-                                );
-                            },
-                        },
-                        {
-                            title: "Requester",
-                            field: "requester",
-                            type: "string",
-                            editable: "never",
-                        },
-                        {
-                            title: "Assignee",
-                            field: "assignee",
-                            type: "string",
-                            editable: "always",
-                        },
-                        {
-                            title: "Updated",
-                            field: "updated",
-                            type: "string",
-                            editable: "never",
-                            render: rowData => <DateTimeText datetime={rowData.updated} />,
-                            filterComponent: DateFilterComponent,
-                        },
-                        {
-                            title: "Path Prefix",
-                            field: "result_path",
-                            type: "string",
-                        },
-                        {
-                            title: "Notes",
-                            field: "notes",
-                            type: "string",
-                            render: rowData => <Note>{rowData.notes}</Note>,
-                            editComponent: props => (
-                                <TextField
-                                    multiline
-                                    value={props.value}
-                                    onChange={event => props.onChange(event.target.value)}
-                                    rows={4}
-                                    fullWidth
-                                />
-                            ),
-                        },
-                        {
-                            title: "ID",
-                            field: "analysis_id",
-                            type: "string",
-                            editable: "never",
-                            defaultFilter: paramID,
-                        },
-                    ]}
+                    columns={columns || []}
```

Comment:
Is `|| []` actually needed here?

Reply:
Nope! This is the relic of an earlier approach. Fixed in [61302ea](https://github.com/ccmbioinfo/stager/pull/567/commits/61302ead558ef578a390c4c18c058dea6f71a2c0)